.pre-commit-config.yaml

```diff
@@ -3,6 +3,7 @@ repos:
     rev: v2.4.0
     hooks:
       - id: flake8
+        args: ["--ignore=E501,W503"]
```

Comment:
Can we comment what these are / why we're ignoring?

Reply:
Yes I will; I would like to centralize these things soon-ish in a pyproject.toml file.